lib/gtfs/reducers/stops.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import type {GtfsStop} from '../../types'
+
+export type StopsState = {
+  fetchStatus: {
+    fetched: boolean,
+    fetching: boolean,
+    error: boolean
+  },
+  data: Array<GtfsStop>
+}
+
+export const defaultState = {
+  routeFilter: null,
+  patternFilter: null,
```

Comment:
These filter properties should be noted in the flow type definition of the state.